src/constants.ts

```diff
@@ -19,3 +19,8 @@ export const DEFAULT_CAMERA_SPEC: CameraSpec = {
     },
     fovDegrees: 75,
 };
+
+export const TEST_CONNECTION_SETTINGS = {
```

Comment:
I feel iffy about this variable name, better ideas welcome

Reply:
I'll propose STAGING_SERVER_SETTINGS ?